jiralib.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+from jira import JIRA
+
+from common import get_envvar
+
+
+class Jira:
+    def __init__(self,
+                 server,
+                 username,
+                 token,
+                 project,
+                 project_id,
+                 component):
+        self.project = project
+        self.project_id = project_id
+        self.server = server
+        self.component = component
+
+        jiraOptions = {"server": self.server}
+        self.jira = JIRA(options=jiraOptions, basic_auth=(username, token))
+
+    def issue_exists(self, repo, id):
+        query = f'project={self.project} AND text ~ "GITHUB:{repo}-{id}"'
+        issues = self.jira.search_issues(query)
+        return len(issues) != 0
+
+    def create_issue(self, repo, issue_id, title, body):
```

Comment:
It will be nicer if the input is just a github issue object, and not having repos and such details involved at this level.